### PR TITLE
feat: empty basket carousel powered by flss in checkout page

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- Empty basket carousel in checkout page powered by FLSS
- Unused code removed

### CODE REVIEW

File: config/dev-local.js [hunk 1]
```diff
@@ -29,6 +29,8 @@ module.exports = merge(base, devVm, {
 	},
 	server: {
 		graphqlUri: 'https://marketplace-api.dk1.kiva.org/graphql',
+		// Alternate graphql url (marketplace-api.dk1.kiva.org is redirected to this url)
+		// graphqlUri: 'https://gateway.development.kiva.org/graphql',
```
Comment (dyersituations) [inline config/dev-local.js:33]: It looks like you can remove these two lines and update line 31 with the new URI.

File: src/util/flssUtils.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ export const FLSS_ORIGIN_LEND_FILTER = 'web:lend-filter';
 export const FLSS_ORIGIN_BP_FUNDED = 'web:bp-funded';
 export const FLSS_ORIGIN_THANKS = 'web:thanks';
 export const FLSS_ORIGIN_PORTFOLIO_OVERVIEW = 'web:portfolio-overview';
+export const FLSS_ORIGIN_CHECKOUT = 'web:empty-checkout';
```
Comment (dyersituations) [inline src/util/flssUtils.js:19]: Did you inform Rob/data-team about the new origin?

File: config/dev-local.js (hunk 1 above)
Comment (roger-in-kiva) [reply config/dev-local.js]: Updated for app and server.

File: src/util/flssUtils.js (hunk 2 above)
Comment (roger-in-kiva) [reply src/util/flssUtils.js:19]: Yeah, Rob and Yariel agreed with this.

File: config/dev-local.js [hunk 3]
```diff
@@ -28,7 +26,7 @@ module.exports = merge(base, devVm, {
 		},
 	},
 	server: {
-		graphqlUri: 'https://marketplace-api.dk1.kiva.org/graphql',
+		graphqlUri: 'https://gateway.development.kiva.org/graphql',
 		sessionUri: 'https://www.dev.kiva.org/start-ui-session',
 		// Alternate session url
 		// sessionUri: 'https://www.development.kiva.org/start-ui-session',
```
Comment (mcstover) [inline config/dev-local.js:32]: Please go ahead and swap in this one and delete the other session uri as it's no longer available.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**